pkg/docker/cockpit-atomic-storage

```diff
@@ -405,15 +410,19 @@ def print_info(data):
         sys.stdout.flush()
 
 def cmd_monitor():
-    # Figure out whether we can manage the pool.  For that, we need
-    # both a volume group to work with, and a recent enough version of
-    # the "atomic" utility.
+    # Figure out whether we can manage the pool.  For that, we
+    # need a recent enough version of the "atomic" utility.
     global can_manage
     can_manage = False
     try:
-        if get_dss_vgroup() != "":
-            with open("/dev/null", "w") as dev_null:
-                can_manage = call([ "atomic", "storage", "--help" ], stdout=dev_null) == 0
+        with open("/dev/null", "w") as dev_null:
+            help_text = check_output([ "atomic", "storage", "modify", "--help" ])
+            if get_dss_vgroup() != "":
+                can_manage = True
+            else:
+                # If the pool ins't yet in a volume group, we need the
+                # --vgroup option to create one.
+                can_manage = "--vgroup" in help_text
     except:
```

Comment:
I would suggest catching the specific type of error that can happen here. Whether it's an IOError or a CalledProcessError